### ISSUE TITLE
I get this error code whenever I try to vote, send tokens, update staking, everything. I basically can use the EOS Voter Wallet:  "permission"."active"}', but does not have signatures for it under a provided delay of 0 ms, provided permissions 

### ISSUE THREAD

**Issue by gregmcgolf** [greymass/eos-voter#267]
Please help

**Comment by gregmcgolf** [comment]
I can't use the wallet at all. I have transferred all my tokens to it, now they are all stuck. Please help

**Comment by aaroncox** [comment]
This generally happens if you end up having the `owner` key imported instead of the `active` key. 

You'll want to remove the local wallet (Click the Greymass logo in the upper right, scroll to the bottom, "Delete Wallet Credentials) and start the import process from scratch again. As you're going through the process - you'll want to import the `active` private key during Step #3 instead of the `owner` key.

**Comment by gregmcgolf** [comment]
OK, here is what I did and still came up with the same issue:
I have two accounts and have the private keys for both.
I  removed the wallet completely, then tried to import the account: I entered the active public key, my account showed up (tpfadminseos) and then I clicked on that and it asked for my private key. I put that in and it said it did not match the public keys. I then did it over and this time I put in my public owner key and again the account name showed up. I clicked on it and it asked for the private key. I input it and once again it said it did not match the public key.
I then went to the explorer to make sure the two keys I had for public owner and active public were correct and they match what is on the explorer.
After that, I then went with the second account that I have. I entered the public key, and the correct account name showed up. I then entered the private key on the next page and voila it worked. All the info showed up on my wallet. I then went to manage wallets and used the import function to add the second account. It only asked for the private key. I entered it and voila, the account name came up and it was accepted and then I have two account names in the wallet with all the coins etc., shown. When I try to take any actions on the first account, everything works, voting, sending tokens, staking, and so forth. Then when I go to the second account (tpfadminseos) and try any of the functions, I keep getting that message about permissions. I did notice that when I imported the account that although it only asked for the private key, it seemed to generate it’s own public keys, but they were not displayed. Is it possible that these are new public keys? I just can’t figure this out, and my funds are all frozen. Thanks for your help.
